site/public/js/resizable-panels.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Initializes Resizables two panels mode
+ * where width from one panel can be given out to another one i.e resizable widths
+ * @param leftPanelSel DOM selector for left panel
```

Comment:
Let's add the JSDoc type hint since you've added the rest of the comment here.
They go like:
```js
@param {JSType} varName Description
```
E.g:
```js
@param {String} foobar string to modify
```

https://jsdoc.app/tags-param.html

Reply:
Added type hintings, thanks!